src/main/scala/com/mwz/sonar/scala/scapegoat/ScapegoatRulesRepository.scala

```diff
@@ -0,0 +1,50 @@
+package com.mwz.sonar.scala
+package scapegoat
+
+import inspections.{Level, ScapegoatInspection}
+
+import org.sonar.api.server.rule.RulesDefinition
+import org.sonar.api.rule.RuleStatus
+import org.sonar.api.rule.Severity
+import org.sonar.api.rules.RuleType
+
+/** Defines a rules repository for the Scapegoat inspections */
+class ScapegoatRulesRepository extends RulesDefinition {
+
+  /** Defines the rules in the repository */
+  override def define(context: RulesDefinition.Context): Unit = {
+    // crete an empty repository
+    val repository =
+      context
+        .createRepository(ScapegoatRulesRepository.RepositoryKey, Scala.LanguageKey)
+        .setName(ScapegoatRulesRepository.RepositoryName)
+
+    // register each scapegoat inspection as a repository rule
+    for (inspection <- ScapegoatInspection.AllScapegoatInspections) {
+      val rule = repository.createRule(inspection.id)
```

Comment:
Since this is going to set the rule key, we probably want this to be prefixed with `scala:scapegoat` instead of the display name below, e.g. `scala:scapegoat:com.sksamuel.scapegoat.inspections.AnyUse`.

Reply:
I'm not sure if thats is necessary, because to search a rule by its internal key, you first need to provide the repository key. So, for me, the prefix would be unnecessary.

Reply:
You're right - the key must be unique in the repository, but not globally unique 👍